docs/CONFIGURATION.md

```diff
@@ -429,8 +429,11 @@ input plugins and before any aggregator plugins.
 Parameters that can be used with any processor plugin:
 
 - **alias**: Name an instance of a plugin.
-- **order**: The order in which the processor(s) are executed. If this is not
-  specified then processor execution order will be random.
+- **order**: The order in which the processor(s) are executed. starting with 1.
+  If this is not specified then processor execution order will be the order in
+  the config. Processors with a defined order will take precedence over those
+  without a defined order. Processors without a defined order will execute
+  first.
```

Comment:
These 2 lines contradict each other. If they take precedence, then they will be executed first, not the ones without an order defined..

Reply:
Oops good catch, I'll update this